pkg/kubectl/proxy_server.go

```diff
@@ -106,8 +106,17 @@ func (f *FilterServer) HandlerFor(delegate http.Handler) *FilterServer {
 	return &f2
 }
 
+// Get the host from a host header
+func HostFromHeader(header string) (host string) {
```

Comment:
Why is this an exported symbol?

better name might be hostFromHostOrHostPort or simply extractHost